subsys/bluetooth/controller/Kconfig

```diff
@@ -77,6 +77,15 @@ config SDC_RX_STACK_SIZE
 	  Size of the receiving thread stack, used to retrieve HCI events and
 	  data from the controller.
 
+config SDC_READ_CHIP_TEMP
```

Comment:
As this command is defined by the Zephyr HCI API, I would propose to add a config there similar to `BT_CTLR_HCI_VS_BUILD_INFO` and `BT_CTLR_TX_PWR_DYNAMIC_CONTROL`. For now I would propose to remove the this new kconfig option

Reply:
Yeah I had it initially in zephyr (have a branch and everything) but then I thought that I'm only adding this so we can optimize out the command when we're not using it in the SDC. 
I thought the linker would do this automatically for the open-source link layer, so we actually only needed this kconfig option for the SDC. So I moved it to sdk-nrf. Another point is that the command is not implemented in the open-source controller yet.
If you think it's still better in to have it in zephyr, I'll move it back.

Reply:
I would prefer to have it in zephyr